pkgs/top-level/python-packages.nix

```diff
@@ -7748,6 +7748,26 @@ in {
     };
   });
 
+
+  htmlmin = buildPythonPackage rec {
```

Comment:
Just curious, is there some criterion for which packages go where?  Or are they all being gradually migrated to python-modules?